tests/unit/jest.config.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const rootDir = '../../';
+const currentDir = '<rootDir>/tests/unit';
+
+module.exports = {
+    clearMocks: true,
+    testRunner: 'jest-circus/runner',
+    transform: {
+        '^.+\\.(ts|tsx)$': 'ts-jest',
+    },
+    verbose: false,
+    coverageDirectory: '<rootDir>/test-results/unit/coverage',
+    displayName: 'unit tests',
+    moduleFileExtensions: ['ts', 'tsx', 'js'],
+    rootDir: rootDir,
+    roots: [currentDir],
+    collectCoverage: true,
+    collectCoverageFrom: [
+        '<rootDir>/src/**/*.{ts,tsx}',
+        '!<rootDir>/tests/**/*',
+        '!<rootDir>/src/**/*.d.ts',
```

Comment:
I don't think we should have any .d.ts files in /src/, so I feel inclined to not configure coverage for them